### DISCUSSION TITLE
개인도메인 사용시 댓글창과 방명록에서 안내문구만 나오지 말고 링크 기능을 지원해 주셧으면 합니다

### DISCUSSION THREAD

**Discussion by bank365** [pronist/hELLO#231] [Ideas]
![Screenshot 2024-06-16 at 05 39 02](https://github.com/pronist/hello/assets/172935667/8d68bf2e-8147-434d-bd30-5b27bc0d05db)
현재는 안내문구만 나오고 있는데요
클릭하면 2차도메인으로 연결되게 했으면 좋겠습니다.
방문자 입장에선 일일이 2차도메인 으로 접속하기가 많이 번거로울것 같습니다

**Comment by pronist** [comment]
@bank365 안녕하세요 :)

제의해주신 기능은 좋은 개선점이라고 생각합니다. 

다음 패치에 반영을 고려해보도록 하겠습니다. :)

**Reply by bank365** [reply]
감사합니다. 꼭 반영 되었음 좋겠습니다^__^
